hack/update-release-parameters.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+
+set -eu
+
+# Accept a file path for param yaml.
+releaseParams=$1
+
+# Get the paths to all files that can be automatically updated for a release.
+updates=$(yq e -o=j -I=0 '.updates[]' "$releaseParams")
+readarray updates <<< "$updates"
+
+for update in "${updates[@]}"; do
+  targetFile=$(yq -e '.targetFile' <<< "$update")
+  templatePath=$(yq -e '.template' <<< "$update")
+  echo -e "Regenerating... $targetFile"
+  draft=$(cat "$templatePath")
+
+  replacements=$(yq e -o=j -I=0 '.replacements[]' <<< "$update")
+  readarray replacements <<< "$replacements"
+  for find in "${replacements[@]//\"/}"; do
+    find=$(tr -d '\n' <<< "$find")
+    replace=$(< "$releaseParams" yq -e ".release-tags.${find}")
+    draft=$(sed -e "s|{{$find}}|$replace|g" <<< "$draft")
+  done
+  echo "$draft" > "$targetFile"
+done
+
+make generate-crd
```

Comment:
@cbandy @tjmoore4 This works, but is this a bad idea?

Reply:
Not that I can think of. I like the idea of the CRD being updated at the same time as everything else.